en/controllers/pagination.rst

```diff
@@ -0,0 +1,294 @@
+Pagination
```

Comment:
This is basically a copy of `controllers/component/pagination` with references to the component removed.